Frameworks/UIKit/UIView.mm

```diff
@@ -274,30 +260,106 @@ - (bool)_processGesturesForTouch:(UITouch*)touch event:(UIEvent*)event touchEven
         }
     }
 
-    // gesture priority list
-    const static id s_gesturesPriority[] = {[UIPinchGestureRecognizer class], [UISwipeGestureRecognizer class],
-                                            [UIPanGestureRecognizer class],   [UILongPressGestureRecognizer class],
-                                            [UITapGestureRecognizer class],   [_UIDMPanGestureRecognizer class] };
+    //
+    // After state transtion, we potentially get active gestures, we should remove any gestures that can not be recognized simultaneously
+    //
+    NSMutableSet* gesturesToRemove = [NSMutableSet set];
```

Comment:
>= [NSMutableSet set]; [](start = 35, length = 21)

seems wasteful to allocate this every time, even if it's not used.  maybe we should use a static that just gets reset as needed?